plugin/src/main/java/com/playmonumenta/scriptedquests/quests/components/actions/dialog/DialogClickableTextEntry.java

```diff
@@ -95,6 +99,8 @@ public DialogClickableTextEntry(String npcName, String displayName, EntityType e
 				mRadius = value.getAsDouble();
 			} else if (key.equals("actions")) {
 				mActions = new QuestActions(npcName, displayName, entityType, delayTicks, value);
+			} else if (key.equals("hover_text")) {
+				mHoverEvent = HoverEvent.showText(MessagingUtils.LEGACY_SERIALIZER.deserialize(value.getAsString().replace("&", "§")));
```

Comment:
Consider doing this the other way around in case the character & is typed literally here.